pkg/apis/configuration/validation/virtualserver.go

```diff
@@ -116,14 +118,18 @@ func validateTLS(tls *v1.TLS, fieldPath *field.Path) field.ErrorList {
 	return allErrs
 }
 
-func validateDos(dos string, fieldPath *field.Path) field.ErrorList {
+func validateDos(isDosEnabled bool, dos string, fieldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 
 	if dos == "" {
 		// valid, dos is not required
 		return allErrs
 	}
 
+	if !isDosEnabled {
+		allErrs = append(allErrs, field.Invalid(fieldPath, dos, "field: 'dos' is used but DOS feature is not enabled"))
```

Comment:
we usually use `field.Forbidden` for the fields with valid values that configured disabled features. Could you update it for consistency?